src/Stratis.Bitcoin.IntegrationTests.Common/TestHelper.cs

```diff
@@ -50,8 +50,7 @@ public static void TriggerSync(CoreNode node)
 
         public static bool IsNodeConnected(CoreNode node)
         {
-            if (node.FullNode.ConnectionManager.ConnectedPeers.Any()) return true;
-            return false;
+            return node.FullNode.ConnectionManager.ConnectedPeers.Any(p => p.IsConnected);
```

Comment:
What about HandShaked? or is it covered in IsConnected?
If yes then push.

Reply:
![image](https://user-images.githubusercontent.com/4638821/42519674-79e0768c-845c-11e8-8b4c-dfe4d033ccaf.png)
Tadaaaa :)